openmc/filter_expansion.py

```diff
@@ -463,3 +463,65 @@ def to_xml_element(self):
         subelement.text = str(self.r)
 
         return element
+
+
+class ZernikeRadialFilter(ZernikeFilter):
+    r"""Score radial Zernike expansion moments in space up to specified order.
```

Comment:
This first line could be more helpful with something like `Score the m=0 (radial variation only) Zernike moments.`